dnf/dnf-main.c

```diff
@@ -497,6 +512,11 @@ main (int   argc,
       else if (opt_install_weak_deps == ARG_FALSE)
         dnf_context_set_install_weak_deps (FALSE);
 
+      if (opt_allow_vendor_change == ARG_TRUE)
+        dnf_context_set_allow_vendor_change (ctx, TRUE);
```

Comment:
It could be replaced by:
```
DnfSack * sack = dnf_context_get_sack(ctx);
if (sack) {
    dnf_sack_set_allow_vendor_change(dnf_context_get_sack(), sack, TRUE)
} else {
    /// report something is not right
}
```

Reply:
But I am nearly sure that at this time sack is NULL.

Reply:
Sack is available after call of `dnf_context_install`.

Reply:
Can I create the sack before that and set flags that the subcommands will use?